Tutorial Contribution System (#956)
polkadot-developers/polkadot-docs: .CONTRIBUTING/tutorials/templates/README.md

# Tutorial Templates

> This folder contains templates for different types of Polkadot tutorials to help you get started quickly and maintain consistency across all documentation.

Reviewer: ```suggestion
This folder includes templates for various types of Polkadot tutorials to help you get started quickly and keep consistency throughout all documentation.
```

## Available Templates

> ### 📦 [Polkadot SDK Template](polkadot-sdk-template.md)

Reviewer: ```suggestion
### 📦 [Polkadot SDK Template](polkadot-sdk-template.md)

```

> For tutorials covering:

Reviewer: ```suggestion
For tutorials covering:

```
